tests/uptane/ats_director.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Just realized that might unnecessary since we never have sign, and the public keys are in the JSON.